src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Connection/IsConnectionsNumberLimitReachedActionEndToEnd.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\back\tests\EndToEnd\Connection;
+
+use Akeneo\Connectivity\Connection\back\tests\EndToEnd\WebTestCase;
+use Akeneo\Connectivity\Connection\Domain\Settings\Model\ValueObject\FlowType;
+use Akeneo\Test\Integration\Configuration;
+use PHPUnit\Framework\Assert;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class IsConnectionsNumberLimitReachedActionEndToEnd extends WebTestCase
+{
+    protected function getConfiguration(): Configuration
+    {
+        return $this->catalog->useMinimalCatalog();
+    }
```

Comment:
```suggestion
    protected function getConfiguration(): ?Configuration
    {
        return null;
    }
```

Trick: If you don't need the fixtures (like default product families etc), you can return null here. Your test will be faster ;)

Reply:
I've tried but null configuration fails so I leave it as it is

Reply:
That's sad. It's not a big deal.